--- FastFlix-5.8.2/fastflix/widgets/windows/profile_window.py	2024-12-22 20:34:49.057691686 -0600
+++ FastFlix-5.8.2/fastflix/widgets/windows/profile_window.py	2024-12-22 20:36:57.851378209 -0600
@@ -15,7 +15,7 @@
 from fastflix.shared import error_message
 from fastflix.encoders.common.audio import channel_list
 
-language_list = sorted((k for k, v in Lang._data["name"].items() if v["pt2B"] and v["pt1"]), key=lambda x: x.lower())
+language_list = sorted((k for k, v in Lang._data["name"].items() if v.get("pt2b") and v.get("pt1")), key=lambda x: x.lower())
 
 logger = logging.getLogger("fastflix")
 
