--- ./main_unix.cpp	2025-02-12 08:56:31.395866275 -0600
+++ ./main_unix.cpp	2025-02-12 08:57:03.037967067 -0600
@@ -1768,7 +1768,7 @@
 											   GTK_MESSAGE_WARNING,
 											   GTK_BUTTONS_NONE,
 											   GetString(title_id), NULL);
-	gtk_message_dialog_format_secondary_text(GTK_MESSAGE_DIALOG(dialog), text);
+	gtk_message_dialog_format_secondary_text(GTK_MESSAGE_DIALOG(dialog), "%s", text);
 	gtk_dialog_add_button(GTK_DIALOG(dialog), GetString(button_id), GTK_RESPONSE_CLOSE);
 	g_signal_connect(dialog, "response", G_CALLBACK(dl_destroyed), NULL);
 	gtk_widget_show(dialog);
